kubernetes/user-svc/helmfile.yaml

```diff
@@ -0,0 +1,47 @@
+repositories:
+
+  - name: bitnami
+    url:  https://charts.bitnami.com/bitnami
+
+
+releases:
+
+  - name: user-postgres
+    namespace: user
+    chart: bitnami/postgresql
+    wait: true
+    set:
+      - name: auth.enablePostgresUser
+        value: true
+      - name: auth.postgresPassword
+        value: helpwave
```

Comment:
please add "TODO" or "FIXME" comments everywhere we use hardcoded secrets, so it's clear this is not the desired state of the code base